bootloader/src/main.rs

```diff
@@ -288,14 +288,17 @@ fn dfu_init() -> LedType {
     /* USB device */
 
     let usb_dev = UsbDeviceBuilder::new(&bus, UsbVidPid(0x2b23, 0x1012))
-        .manufacturer("Red Hat Inc.")
-        .product("Jumpstarter DFU Mode")
-        .serial_number(get_serial_str())
+        .strings(&[
+            StringDescriptors::new(LangID::EN)
+            .manufacturer("Red Hat Inc.")
+            .product("Jumpstarter DFU Mode")
```

Comment:
```suggestion
            .product("Dutlink DFU Mode")
```

not related to this change, but since we are at it...